packages/gafl-webapp-service/src/handlers/__tests__/__snapshots__/page-handler.spec.js.snap

```diff
@@ -41,6 +42,7 @@ exports[`The page handler function sets the value of pageData with displayAnalyt
         "analyticsSelected": false,
         "backRef": null,
         "displayAnalytics": true,
+        "gtmContainerId": null,
```

Comment:
Please could we set gtmContainerId to a default value in the test so that we don't end up with a null value in the snapshot test?

Reply:
Fixed – it defaults to false instead now